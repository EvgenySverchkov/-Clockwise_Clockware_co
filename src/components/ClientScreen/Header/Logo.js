import React from "react";
import {Link} from "react-router-dom";
import PropTypes from "prop-types";

const Logo = ({children}) => (
    <Link to="/client" className="navbar-brand">
          {children}
    </Link>
);

Logo.propTypes = {
    children: PropTypes.node.isRequired
}

export default Logo;