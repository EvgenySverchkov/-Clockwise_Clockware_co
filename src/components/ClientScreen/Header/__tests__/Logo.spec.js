import React from "react";
import {create} from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";

import Logo from "../Logo";

it("Test of <Logo /> on client side", ()=>{
    const component = create(
        <MemoryRouter initialEntries={["/"]}>
            <Logo>
                I'm LOGO
            </Logo>
        </MemoryRouter>
    );
    expect(component.toJSON()).toMatchSnapshot();
});