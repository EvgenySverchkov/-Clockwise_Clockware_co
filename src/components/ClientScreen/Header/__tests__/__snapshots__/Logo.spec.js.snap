// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of <Logo /> on client side 1`] = `
<a
  className="navbar-brand"
  href="/client"
  onClick={[Function]}
>
  I'm LOGO
</a>
`;
